subgraphs/burrow/src/handlers/farm.ts

```diff
@@ -0,0 +1,135 @@
+import {
+  JSONValueKind,
+  log,
+  BigInt,
+  BigDecimal,
+} from "@graphprotocol/graph-ts";
+import { getOrCreateMarket } from "../helpers/market";
+import { getOrCreateRewardToken, getOrCreateToken } from "../helpers/token";
+import { BD_ZERO } from "../utils/const";
+import { EventData } from "../utils/type";
+
+/**
+ * Add farm tokens
+ * @param event EventData
+ * @returns
+ */
+export function handleAddAssetFarmReward(event: EventData): void {
+  const data = event.data;
+
+  const farm_id = data.get("farm_id");
+  if (!farm_id) {
+    log.warning("handleAddAssetFarmReward() :: farm_id not found", []);
+    return;
+  }
+  if (farm_id.kind != JSONValueKind.OBJECT) {
+    log.info("handleAddAssetFarmReward() :: Incorrect type farm_id {}", [
+      farm_id.kind.toString(),
+    ]);
+    return;
+  }
+  const farm_id_obj = farm_id.toObject();
+  let farm_id_asset = farm_id_obj.get("Supplied");
+  let farmType = "DEPOSIT";
+  if (!farm_id_asset) {
+    farm_id_asset = farm_id_obj.get("Borrowed");
+    farmType = "BORROW";
+    if (!farm_id_asset) {
+      log.warning("handleAddAssetFarmReward() :: farm_id_asset not found", []);
+      return;
+    }
+  }
+  const farm = farm_id_asset.toString();
+
+  const reward_token_id = data.get("reward_token_id");
+  if (!reward_token_id) {
+    log.warning("handleAddAssetFarmReward() :: reward_token_id not found", []);
+    return;
+  }
+  const reward_token = reward_token_id.toString();
+
+  const new_reward_per_day_ = data.get("new_reward_per_day");
+  if (!new_reward_per_day_) {
+    log.warning(
+      "handleAddAssetFarmReward() :: new_reward_per_day not found",
+      []
+    );
+    return;
+  }
+  const new_reward_per_day = new_reward_per_day_.toString();
```

Comment:
The standard for variable naming is camelCase

Reply:
Please feel free to make changes as you see fit